src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.iOSAndroid.cs

```diff
@@ -20,7 +20,7 @@ public partial class ListViewBase
 		/// The native ListView type which is providing the implementation. This will be null if a panel other than <see cref="ItemsStackPanel"/>
 		/// or <see cref="ItemsWrapGrid"/> is being used.
 		/// </summary>
-		internal NativeListViewBase NativePanel => InternalItemsPanelRoot as NativeListViewBase;
+		public NativeListViewBase NativePanel => InternalItemsPanelRoot as NativeListViewBase;
```

Comment:
Let's make this a helper for that property or `SmoothScrollToPosition`, rather than a method exposed on `ListViewBase`, also the API is definitely not easy to consume on `SmoothScrollToPosition`. Can you add a runtime test?